app/routes/wallet/sd_jws.py

```diff
@@ -65,16 +94,30 @@ async def sign_sd_jws(
     "/verify",
     response_model=SDJWSVerifyResponse,
     summary="Verify SD-JWS",
-    description="""
-Verify Select Disclosure for JWS (SD-JWS)
-
-See https://www.ietf.org/archive/id/draft-ietf-oauth-selective-disclosure-jwt-07.html for the SD-JWT / SD-JWS spec.
-""",
 )
 async def verify_sd_jws(
     body: SDJWSVerifyRequest,
     auth: AcaPyAuth = Depends(acapy_auth_from_header),
 ) -> SDJWSVerifyResponse:
+    """
+    Verify Select Disclosure for JWS (SD-JWS)
+    ---
+
+    There are some extra keys needed to reveal the selective disclosure attributes.
+    These keys are included in the signed SD-JWS endpoint.
```

Comment:
I'm not sure what this means ... There are extra keys needed? And they are included in other endpoint?  wut, this endpoint just takes a sd-jws string as input

Similar to verify JWS endpoint, we can just include some words about "this validates authenticity of a SD-JWS string"